src/Symfony/Component/Console/Tests/Command/CommandTest.php

```diff
@@ -442,48 +442,6 @@ public function testCommandAttribute()
         $this->assertSame(['f'], $command->getAliases());
     }
 
-    /**
-     * @group legacy
-     */
-    public function testDefaultNameProperty()
-    {
-        $this->expectDeprecation('Since symfony/console 6.1: Relying on the static property "$defaultName" for setting a command name is deprecated. Add the "Symfony\Component\Console\Attribute\AsCommand" attribute to the "Symfony\Component\Console\Tests\Command\MyCommand" class instead.');
-
-        $this->assertSame('my:command', MyCommand::getDefaultName());
-    }
-
-    /**
-     * @group legacy
-     */
-    public function testDefaultDescriptionProperty()
-    {
-        $this->expectDeprecation('Since symfony/console 6.1: Relying on the static property "$defaultDescription" for setting a command description is deprecated. Add the "Symfony\Component\Console\Attribute\AsCommand" attribute to the "Symfony\Component\Console\Tests\Command\MyCommand" class instead.');
-
-        $this->assertSame('This is a command I wrote all by myself', MyCommand::getDefaultDescription());
-    }
-
-    /**
-     * @group legacy
-     */
-    public function testStaticDefaultProperties()
-    {
-        $command = new MyCommand();
-
-        $this->assertSame('my:command', $command->getName());
-        $this->assertSame('This is a command I wrote all by myself', $command->getDescription());
-    }
-
-    public function testAttributeOverridesProperty()
```

Comment:
this one is not legacy

Reply:
That's right, but now that `$defaultName` and `$defaultDescription` properties are removed, it feels irrelevant to me to have this test. I guess this test should have been marked as legacy as well when the properties were deprecated, or maybe am I missing something?

Reply:
That's right, but now that `$defaultName` and `$defaultDescription` are removed, it feels irrelevant to me to have this test. I guess this should have been marked as legacy as well when the properties were deprecated? Or maybe am I missing something?

Reply:
That's reverted and updated :+1: